docker/single-server/prepare_data.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+#
+set -x
+
+cd /opt/zextras/bin || exit
+
+DOMAIN="carbonio-system.svc.cluster.local"
+# Update zextras password
+carbonio prov sp zextras password
+
+# Create test user
+carbonio prov ca test@"${DOMAIN}" password
+
+# Create admin user
+carbonio prov ca admin@"${DOMAIN}" password zimbraIsAdminAccount TRUE
+
+# Create shared account
+carbonio prov ca shared@"${DOMAIN}" password displayName "Shared"
```

Comment:
```suggestion
carbonio prov ca shared@"${DOMAIN}" password displayName "Shared account"
```
Just to avoid confusion in UI :wink: